### ISSUE TITLE
[Bug, Xnet]: XNet controller NBT causes big packet on FTB Jar openGui

### ISSUE THREAD

**Issue by MiningPickaxe** [FTBTeam/FTB-Modpack-Issues#278]
### Modpack

FTB Ocean Block

### Modpack version

1.7.0

### Log Files

https://paste.feed-the-beast.com/view/dfc9f958

### Describe the bug

I'm playing on a Server. When right-clicking a tempered jar no GUI opens and the server logs an error which is also included in the log at the paste. Tested with Version 1.6.0, also persists with 1.7.0. Tried a fresh install of the server only keeping the world and server.properties, same result.

### Steps to reproduce

1. right click a Tempered Jar

### Expected behaviour

Should open the GUI

**Comment by Raidobw2** [comment]
Hello, I've tried reproducing the error with a brand new client and server on 1.7.0 but it works fine for me :/
![tempered_jar](https://user-images.githubusercontent.com/17010632/140876027-81eb1ebc-154b-4894-a1d9-7914a1b7778d.gif)
In your server log I notice the line: `Invalid PacketBuffer for openGui, found 109288 bytes`.
It looks like that packet is big. A similar issue is found at: refinedmods/refinedstorage#2378.

Can you try to open the Tempered Glass Jar with nothing in your hand at all, or in a different chunk?

**Comment by Raidobw2** [comment]
Might be due to another mod's NBT. To be investigated upon receiving a reply.

**Comment by MiningPickaxe** [comment]
I tried opening it with an empty hand, didn't work. Different chunk waaay outside my base, didn't work.
What did work: Removing an XNet Controller from my inventory. It seems like the Controller with its stored configuration has too much data to fit into that packet. 
I did a /kubejs hand to get the data of that controller just in case you are interested: https://paste.feed-the-beast.com/view/72e37f3b
Its rather large at roughly 84.5 KB. 

My guess is that when clicking the jar it gets the entire players inventory and sends it to the client again to display it on the left side of the GUI with the avaible items? Maybe there should be some error handling there with huge NBT data of items?

**Comment by Raidobw2** [comment]
Yeah it's super helpful thank you. I'll be able to take it from there, thanks for your help!

**Comment by Raidobw2** [comment]
File issue to XNet just above ^

@LatvianModder do you need a report or you think it's fully XNet?
TL;DR: Tempered Jar fails to openGui server side because XNet Controller in inventory weights 85KB (NBT), causing `Invalid PacketBuffer for openGui, found 109288 bytes` on right-click.

**Comment by NecoAngelus** [comment]
I encountered a similar error, but I had no X-Net Controller in my inventory. I had my Inventory filled with full Shulker Chests. 
After I put all Shulker Chests in a Chest it worked again.

**Comment by Raidobw2** [comment]
If possible, could you hold that filled "shulker chest" in your hand and type `/kubejs hand`? This would give you the contents of it in the form of text, which then you could paste here with https://paste.feed-the-beast.com/ for example.
(You can click on the text in the chat to copy it).
This would help me figure out if it's a FTB Jar Mod issue instead.
After thinking it through, it might be, since the Tempered Jar allows crafting from your inventory, so it might "scan" for items.

**Comment by NecoAngelus** [comment]
I tried everything, in Single player as well on the Server. After a restart of the Server the issue disappeared.
No one has that issue any longer. If I encounter it again I will make the export of the logs. 
Maybe it is sometimes a Server Hick up, like the Server Script for the Mekanism Suit. One reload of the script fixes that to 😁

**Comment by Raidobw2** [comment]
That's quite a peculiar issue. Thanks for the follow up. We'll manage it from there, there's a possibility we'll close this issue soon as well since it's resolved for you.

**Comment by Sunekaer** [comment]
Should be fixed in the newest version of the pack, if not let me know
